common/plugins/com.liferay.ide.ui/plugin.xml

```diff
@@ -182,8 +182,8 @@
             id="com.liferay.ide.ui.workspace.launchWorkspace"
             name="Launch Workspace">
             <commandParameter
-                  id="com.liferay.ide.ui.workspace.launchWorkspace.workspaceLocation"
-                  name="Workspace Location"
+                  id="workspaceLocation"
```

Comment:
Why did you need to update this?  I'm not opposed to these values but just wondering why you needed to change them?